src/sniffer.c

```diff
@@ -2524,12 +2524,15 @@ static int ProcessSessionTicket(const byte* input, int* sslBytes,
                                 SnifferSession* session, char* error)
 {
     word16 len;
+
+#ifdef WOLFSSL_TLS13
```

Comment:
I don't get any compiler error with "--disable-tls13 --enable-sniffer" on OSX.  Can you clarify the build error you are seeing that made this change necessary?

Reply:
It looks like compiler dependent.

gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)

$ ./configure --disable-tls13 --enable-sniffer
===
src/sniffer.c: In function ‘ProcessSessionTicket’:
src/sniffer.c:2522:14: error: variable ‘ssl’ set but not used [-Werror=unused-but-set-variable]
     WOLFSSL* ssl;
              ^~~
src/sniffer.c: In function ‘DoHandShake’:
src/sniffer.c:3568:13: error: label ‘exit’ used but not defined
             goto exit;
===